templates/default.html

```diff
@@ -13,7 +13,14 @@
       </a>
       <nav role="navigation" aria-label="Main">
         <a role="menuitem" href="/download/">Download</a>
-        <a role="menuitem" href="/docs/">Documentation</a>
+        <div class="has-submenu">
+          <a role="menuitem" href="/docs/">Documentation</a>
+            <div class="submenu">
+              <a role="menuitem" href="/commands/">Command Reference</a>
+              <a role="menuitem" href="/docs/topics/">Documentation by Topic</a>
+              <a role="menuitem" href="/technical-blog-posts/">Technical Blog Posts</a>
```

Comment:
I don't consider technical blog posts to be documentation, and think they should have a separate section.

Reply:
I agree! Is there a need to distinguish between blog vs Technical blog? I think they can both reside under community>blogs.

Reply:
They're all three different things in my book. 

This came from a recent meeting I was in where an attendee remarked that people wanting to learn need to have a single place to find things instead of having to hunt through multiple sections. I definitely see this point.

Would it work better if we had "Learn" as the heading menu?

Reply:
Blogs are usually point in time content and they should not be treated as source of truth for documentation. Its definitely a good starting point for learning though.

Reply:
@bbarani This is a taxonomy of blog posts, so it would show up in Community > Blogs (along side things that are non-technical) as well as here.

The taxonomy lets us further categorize out the blog posts into topic areas. There is only one technical blog post right now, but it would expand out naturally as more topic areas are covered.

![Screenshot 2024-06-17 at 1 07 34 PM](https://github.com/valkey-io/valkey-io.github.io/assets/1152927/680d5d4a-a8b9-4946-a53b-c78924f6d89c)

Reply:
Got it! I was wondering if we can do the taxonomy changes within /blog/ page ( Display different sections for community blogs vs technical blogs) rather than having separate header.

Reply:
I've changed this from 'Technical Blog Posts' to 'Tutorials' to better fit what I was trying to get at (and to address the updating, non-point in time nature of how these need to be treated.

@bbarani 

> I was wondering if we can do the taxonomy changes within /blog/ page ( Display different sections for community blogs vs technical blogs) rather than having separate sub menu for technical blog.

I think this might be a confusion with a post category. Taxonomies are different: they aren't a function of the blog - any content (post or not) could be sorted into a taxonomy, so having as a function of `/blog/` doesn't make a lot of sense.

Let me give you a more concrete example: I could create a list of links as a content type in Zola and add the `tutorials` taxonomy to this and it would be listed along side the blog post marked in this example even though those links aren't blog post. It's quite a bit more powerful than just a blog category.